### PR TITLE
Update RSEQC and fixes MultiQC bug

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

This PR  includes the following changes:
- Modifies [junction_annotation.xml](https://github.com/galaxyproject/tools-iuc/compare/master...gallardoalba:tools-iuc:Update_rseqc_fix_multiqc?expand=1#diff-7307f3a620c331cf5f12200efe5ae5cbf0f93819b137ea0080dd224a6a5c94cd) in order to include a missing output required for plotting the results in MultiQC. 
- It improves the labels of the different RSeQC tools
- It fixes [a bug in MultiQC](https://github.com/galaxyproject/tools-iuc/pull/4991/files#diff-132ba9b18dd57a2c00b2158a0e5aae026127589126fbb66eb5ddc74385f0fb96R221-R225) (previously it could not plot the results from junction_saturation). In addition, it improves the labels and include a new test.

### CODE REVIEW

**Comment by gallardoalba** [comment]
The current error seems to not be related with the update:

`  File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/galaxy/tool_util/linters/general.py", line 42, in lint_general
    elif isinstance(parsed_version, packaging.version.LegacyVersion):
AttributeError: module 'packaging.version' has no attribute 'LegacyVersion'`

**Comment by mvdbeek** [comment]
Right, https://github.com/galaxyproject/galaxy/pull/15148 will fix that

File: tools/rseqc/junction_annotation.xml [hunk 1]
```diff
@@ -25,6 +26,7 @@
             --out-prefix output
             --min-intron ${min_intron}
             --mapq ${mapq}
+        2> stats.txt
```
Comment (mvdbeek) [inline tools/rseqc/junction_annotation.xml:29]: This will swallow all of the stderr, making this very hard to debug. Can you use `tee` here?

**Review by mvdbeek — COMMENTED** [review]
Looking good, and tests are passing too now :)

File: tools/rseqc/junction_annotation.xml (hunk 1 above)
Comment (gallardoalba) [reply tools/rseqc/junction_annotation.xml:29]: Sure, thanks!

File: tools/rseqc/junction_annotation.xml [hunk 2]
```diff
@@ -25,6 +26,7 @@
             --out-prefix output
             --min-intron ${min_intron}
             --mapq ${mapq}
+        2> >(tee stats.txt)
```
Comment (bernt-matthias) [inline tools/rseqc/junction_annotation.xml:29]: Maybe `2> >(tee -a stats.txt >&2)`

Otherwise the tools stderr will end up on stdout

Comment (gallardoalba) [reply tools/rseqc/junction_annotation.xml]: Done, thanks a lot!

**Review by bgruening — APPROVED** [review]

**Pull request merged**